source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc

```diff
@@ -19,3 +19,5 @@ For example, `10-timezone.properties` overrides the default file, `00-timezone.p
 +
 General:: Time zones used for non-Windows operating system types, must follow the standard time zone format for example, `Etc/GMT` or `Asia/Jerusalem`.
 Windows:: Time zones specifically supported on link:https://docs.microsoft.com/en-us/previous-versions/windows/embedded/ms912391(v=winembedded.11)?redirectedfrom=MSDN[Windows] for example, `GMT Standard Time` or `Israel Standard Time`.
++
```

Comment:
This line is not necessary.

Reply:
I think its needed with the suggested change

Reply:
Ah, yes, you're right. ;D